src/Header.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Created by ttnd on 27/12/16.
+ */
+import React from 'react'
+
+export default class Header extends React.Component {
+  constructor() {
+    super();
```

Comment:
pass props as a variable to super as a best practice to avoid having this.props as undefined in constructor
constructor(props) { super(props) }